pkg/apis/core/types_shoot.go

```diff
@@ -504,7 +504,14 @@ type ClusterAutoscaler struct {
 	MaxNodeProvisionTime *metav1.Duration
 	// MaxGracefulTerminationSeconds is the number of seconds CA waits for pod termination when trying to scale down a node (default: 600).
 	MaxGracefulTerminationSeconds *int32
+	// StartupTaints specifies a list of taint keys to ignore in node templates when considering to scale a node group.
+	// Cluster Autoscaler treats nodes tainted with startup taints as unready, but taken into account during scale up logic, assuming they will become ready shortly.
+	StartupTaints []string
+	// StatusTaints specifies a list of taint keys to ignore in node templates when considering to scale a node group.
+	// Cluster Autoscaler internally treats nodes tainted with status taints as ready, but filtered out during scale up logic.
+	StatusTaints []string
 	// IgnoreTaints specifies a list of taint keys to ignore in node templates when considering to scale a node group.
+	// Ignore taints are now deprecated and treated as startup taints.
```

Comment:
We should forbid setting this field for Kubernetes 1.32 and higher `Shoot`s to make sure we can eventually get rid of it in the `Shoot` API. Please adapt the doc string and implement this kind of validation.